lib/css_parser/version.rb

```diff
@@ -1,3 +1,3 @@
 module CssParser
-  VERSION = "1.3.6".freeze
+  VERSION = "1.3.7".freeze
```

Comment:
FYI version bumps should not be done in PRs, lots of merge conflicts if multiple people are doing it + whoever releases should decide on the version number